### PR TITLE
Update to new menu location and new setting names

### PR DESCRIPTION
**Pull request by get-itips**
closes https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/7906

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/set-up-guests.md [hunk 1]
```diff
@@ -48,22 +48,23 @@ description: Learn about how to turn on or turn off the guest access feature in
...
       - **Giphy content rating** –  Select a rating from the drop-down list:
           - **Allow all content** - Guests will be able to insert all Giphys in chats, regardless of the content rating.
           - **Moderate** - Guests will be able to insert Giphys in chats, but will be moderately restricted from adult content.
           - **Strict** – Guests can insert Giphys in chats, but will be restricted from inserting adult content.
-      - **Use memes in conversations** - Turn this setting **On** to allow guests to use Memes in conversations.
-      - **Use Stickers in conversations** – Turn this setting **On** to allow guests to use stickers in conversations.
+      - **Memes in conversations** - Turn this setting **On** to allow guests to use Memes in conversations.
+      - **Stickers in conversations** – Turn this setting **On** to allow guests to use stickers in conversations.
+      - **Immersive reader for messages** - Turn this setting **On** to let guests to view messages in Microsoft Immersive Reader.
```
Comment (JohanFreelancer9) [inline Teams/set-up-guests.md:67]:
```suggestion
      - **Immersive reader for messages** - Turn this setting **On** to allow guests to view messages in Microsoft Immersive Reader.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@MikePlumleyMSFT PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**